client/src/index.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
+import App from './components/App';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Can we avoid this by using CDN directly in public/index.html

Reply:
We can, but this helps me import bootstrap classes into my react app. (For intellisense/autocomplete)